app/controllers/admin/export/document_controller.rb

```diff
@@ -1,8 +1,44 @@
+require "#{Rails.root}/lib/whitehall/exporters/documents_info_exporter"
```

Comment:
Do we need to require this? it seems to be in the eager load path: https://github.com/alphagov/whitehall/blob/26e2e9eecd862af329cd85184e93cfe9c1932789/config/application.rb#L18-L20

If we did need to require it we'd have lib in the include path so just `require "whitehall/exporters/documents_info_exporter"` would work.

Reply:
We don't actually need require as it turns out, I just needed to add the correct namespaces to the class / class calls.